### PR TITLE
more artifacts, fewer coordinates

### PR DESCRIPTION
**Pull request by elharo**
@garrettjonesgoogle Clean up code with less back and forth conversion of artifacts to and from coordinate strings

### CODE REVIEW

File: dashboard/src/main/java/com/google/cloud/tools/opensource/dashboard/DashboardMain.java [hunk 1]
```diff
@@ -126,59 +131,56 @@ private static void generateReports(Configuration configuration, Path output,
           || managed.getArtifactId().equals("google-cloud-contrib")) {
         continue;
       }
-      String coords = Artifacts.toCoordinates(managed);
-      if (!result.contains(coords)) {
-        result.add(coords);
+      // String coords = Artifacts.toCoordinates(managed);
```
Comment (garrettjonesgoogle) [inline dashboard/src/main/java/com/google/cloud/tools/opensource/dashboard/DashboardMain.java]: delete dead code?

File: dashboard/src/main/java/com/google/cloud/tools/opensource/dashboard/DashboardMain.java [hunk 2]
```diff
@@ -65,45 +64,51 @@ public static void main(String[] args)
...
+    List<Artifact> managedDependencies = readBom(bom);
+    
+    generateDashboard(configuration, output, managedDependencies);
+    generateReports(configuration, output, managedDependencies);
     
     return output;
   }
 
-  private static Configuration configure() {
+  private static Configuration configureFreemarker() {
```
Comment (garrettjonesgoogle) [inline dashboard/src/main/java/com/google/cloud/tools/opensource/dashboard/DashboardMain.java]: What is Freemarker?

Comment (elharo) [reply dashboard/src/main/java/com/google/cloud/tools/opensource/dashboard/DashboardMain.java]: The template engine we use to generate the dashboard HTML.

File: dashboard/src/main/java/com/google/cloud/tools/opensource/dashboard/DashboardMain.java (hunk 1 above)
Comment (elharo) [reply dashboard/src/main/java/com/google/cloud/tools/opensource/dashboard/DashboardMain.java]: done

**Pull request merged**